### ISSUE TITLE
SBNCode doesn't build standalone

### ISSUE THREAD

**Issue by ikatza** [SBNSoftware/sbncode#84]
Following the split of `sbncode/sbnana` I can't build `sbncode` on its own. 

I'm getting a few errors like this:

```bash
[57/127] Building CXX object sbncode/sbncode/CAFMaker/CMakeFiles/CAFMaker.dir/FillFlashMatch.cxx.o
FAILED: sbncode/sbncode/CAFMaker/CMakeFiles/CAFMaker.dir/FillFlashMatch.cxx.o
/cvmfs/larsoft.opensciencegrid.org/products/gcc/v8_2_0/Linux64bit+3.10-2.17/bin/g++  -DCAFMaker_EXPORTS -DNDEBUG -I/sbnd/app/users/icaza/work/sbncode/srcs -I/sbnd/app/users/icaza/work/sbncode/srcs/sbncode -Isbncode -I/cvmfs/larsoft.opensciencegrid.org/products/cppunit/v1_14_0/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/clhep/v2_4_1_2/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/cetlib_except/v1_04_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/sqlite/v3_26_00_00/Linux64bit+3.10-2.17/include -I/cvmfs/larsoft.opensciencegrid.org/products/tbb/v2019_3/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/hep_concurrency/v1_04_01/slf7.x86_64.e19.prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/cetlib/v3_10_00/slf7.x86_64.e19.prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/fhiclcpp/v4_11_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/messagefacility/v2_06_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/range/v3_0_4_0/include -I/cvmfs/larsoft.opensciencegrid.org/products/canvas/v3_09_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/root/v6_18_04d/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/larcoreobj/v09_01_00/include -I/cvmfs/larsoft.opensciencegrid.org/products/larcorealg/v09_00_02/include -I/cvmfs/larsoft.opensciencegrid.org/products/art/v3_05_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/canvas_root_io/v1_05_02/include -I/cvmfs/larsoft.opensciencegrid.org/products/art_root_io/v1_03_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/larcore/v09_01_00/include -I/cvmfs/larsoft.opensciencegrid.org/products/dk2nudata/v01_08_00g/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/nusimdata/v1_21_05/include -I/cvmfs/larsoft.opensciencegrid.org/products/lardataobj/v09_00_07/include -I/cvmfs/larsoft.opensciencegrid.org/products/lardataalg/v09_03_06/include -I/cvmfs/larsoft.opensciencegrid.org/products/postgresql/v9_6_11b/Linux64bit+3.10-2.17-p372/include -I/cvmfs/larsoft.opensciencegrid.org/products/fftw/v3_3_8a/Linux64bit+3.10-2.17-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/lardata/v09_01_09/include -I/cvmfs/larsoft.opensciencegrid.org/products/libwda/v2_28_0/Linux64bit+3.10-2.17/include -I/cvmfs/larsoft.opensciencegrid.org/products/larevt/v09_01_06/include -I/cvmfs/larsoft.opensciencegrid.org/products/marley/v1_1_1e/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/nutools/v3_07_05/include -I/cvmfs/larsoft.opensciencegrid.org/products/xerces_c/v3_2_2/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/nug4/v1_05_04/include -I/cvmfs/larsoft.opensciencegrid.org/products/genie/v3_00_06i/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/dk2nugenie/v01_08_00l/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/nugen/v1_10_12/include -I/cvmfs/larsoft.opensciencegrid.org/products/nurandom/v1_04_05/include -I/cvmfs/larsoft.opensciencegrid.org/products/log4cpp/v1_1_3b/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/larsim/v09_08_00/include -I/cvmfs/larsoft.opensciencegrid.org/products/eigen/v3_3_5/include/eigen3 -I/cvmfs/larsoft.opensciencegrid.org/products/larreco/v09_04_10/include -I/cvmfs/larsoft.opensciencegrid.org/products/nuevdb/v1_02_09/include -I/cvmfs/larsoft.opensciencegrid.org/products/lareventdisplay/v09_00_20/include -I/cvmfs/larsoft.opensciencegrid.org/products/larexamples/v09_00_19/include -I/cvmfs/larsoft.opensciencegrid.org/products/larana/v09_00_20/include -I/cvmfs/larsoft.opensciencegrid.org/products/protobuf/v3_11_2a/Linux64bit+3.10-2.17-e19/include -I/cvmfs/larsoft.opensciencegrid.org/products/tensorflow/v1_12_0c/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/trtis_clients/v19_11b/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/larrecodnn/v09_04_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/pandora/v03_16_00/Linux64bit+3.10-2.17-e19-prof/include -I/cvmfs/larsoft.opensciencegrid.org/products/larpandoracontent/v03_22_04/include -I/cvmfs/larsoft.opensciencegrid.org/products/larpandora/v09_04_01/include -I/cvmfs/larsoft.opensciencegrid.org/products/TRACE/v3_16_00/include -I/cvmfs/larsoft.opensciencegrid.org/products/artdaq_core/v3_06_03/include -I/cvmfs/sbnd.opensciencegrid.org/products/sbnd/sbnobj/v09_10_00/include -I/sbnd/app/users/icaza/work/sbncode/srcs/sbncode/sbncode/CAFMaker -I/cvmfs/larsoft.opensciencegrid.org/products/ifdhc/v2_5_7/Linux64bit+3.10-2.17-e19-p372-prof/inc -I/cvmfs/larsoft.opensciencegrid.org/products/ifdh_art/v2_10_07/slf7.x86_64.e19.prof/inc -isystem /cvmfs/larsoft.opensciencegrid.org/products/boost/v1_70_0/Linux64bit+3.10-2.17-e19-prof/include -g -gdwarf-4 -O3 -fno-omit-frame-pointer -std=c++98 -Werror -Wno-error=deprecated-declarations -pedantic -Wno-unused-local-typedefs -Wno-undefined-var-template -I /include/c++/v1 -std=c++17 -Wall -Werror=return-type -fPIC -MD -MT sbncode/sbncode/CAFMaker/CMakeFiles/CAFMaker.dir/FillFlashMatch.cxx.o -MF sbncode/sbncode/CAFMaker/CMakeFiles/CAFMaker.dir/FillFlashMatch.cxx.o.d -o sbncode/sbncode/CAFMaker/CMakeFiles/CAFMaker.dir/FillFlashMatch.cxx.o -c /sbnd/app/users/icaza/work/sbncode/srcs/sbncode/sbncode/CAFMaker/FillFlashMatch.cxx
In file included from /sbnd/app/users/icaza/work/sbncode/srcs/sbncode/sbncode/CAFMaker/FillFlashMatch.cxx:7:
/sbnd/app/users/icaza/work/sbncode/srcs/sbncode/sbncode/CAFMaker/FillFlashMatch.h:14:10: fatal error: sbnanaobj/StandardRecord/StandardRecord.h: No such file or directory
 #include "sbnanaobj/StandardRecord/StandardRecord.h"
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

where crucially the path to `StandardRecord.h` is not included.

**Comment by cjbackhouse** [comment]
Do you have a version of sbnanaobj set up at all?
If not, shouldn't we have got an error somewhere saying we couldn't find it?

**Comment by cjbackhouse** [comment]
Does adding a `find_ups_product(sbnanaobj` to `sbncode/CMakeLists.txt` help?

**Comment by ikatza** [comment]
The ups product is set:

```
ups active  | grep sbnanaobj
sbnanaobj         v09_15_00       -f Linux64bit+3.10-2.17 -q e19:prof        -z /cvmfs/sbnd.opensciencegrid.org/products/sbnd
```

Adding `find_ups_product(sbnanaobj` to `sbncode/CMakeLists.txt` does help. It can now build all the way to the end.

Thanks.

**Comment by cjbackhouse** [comment]
Could you make a PR?

**Comment by ikatza** [comment]
https://github.com/SBNSoftware/sbncode/pull/85

**Comment by cjbackhouse** [comment]
Fixed by #85 (I hope)
